sormas-api/src/main/resources/strings.properties

```diff
@@ -1229,6 +1229,8 @@ messageVaccinationNotRelevantForContact = This vaccination is not relevant for t
 messageVaccinationNoDateNotRelevantForContact = This vaccination is not relevant for this contact because it does not have a vaccination date.
 messageVaccinationNotRelevantForEventParticipant = This vaccination is not relevant for this event participant because the vaccination date is after the event date or event report date.
 messageVaccinationNoDateNotRelevantForEventParticipant = This vaccination is not relevant for this event participant because it does not have a vaccination date.
+messageAcceptRequestToNavigate = The request is not yet accepted. Please accept it to be able to navigate.
```

Comment:
Please change this to:  The request is not yet accepted. You have to accept it first to be able to navigate there.